src/tree/transformer.ts

```diff
@@ -213,11 +213,11 @@ const getMetadata = (element: Source): ComponentMetadata => {
 };
 
 const getComponentDirectives = (metadata: ComponentMetadata): Array<string> => {
-  if (metadata == null || metadata.directives == null) {
+  if (metadata == null || metadata['directives'] == null) {
     return [];
```

Comment:
Directives are no longer part of component definitions as of RC6. Is this for backward compatibility?